### DISCUSSION TITLE
IKEA LED2106R3 still says "Not available" in device list

### DISCUSSION THREAD

**Discussion by stekel01** [Koenkk/zigbee2mqtt#20046] [Devices]
Hello,

According to https://github.com/Koenkk/zigbee2mqtt/discussions/19760 the IKEA LED2106R3 should be supported in the latest  Z2M version. Unfortunately the device is still not supported in my case. I tried re-joining the device, but no success. What I notice is that the device name differs from the one in the link (see screenshot). TRADFRI_bulb_GU10_WS_345lm vs TRADFRI bulb GU10 **WW** 345lm

Is there anything I can do so that my device is supported as well?

Current version: 1.34.0-1
![Screenshot 2023-12-05 172310](https://github.com/Koenkk/zigbee2mqtt/assets/150069830/06e47a65-e645-4c61-8239-b3a4a94e2ed3)

**Comment by Koenkk** [comment]
Does this device support color temperature?

**Comment by stekel01** [comment]
Hello, thanks for your answer.

It supports brightness control and color temperature control.

**Reply by Koenkk** [reply]
Added!

Changes will be available in the [dev branch](https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html) in a few hours from now.

**Reply by stekel01** [reply]
Thank you! Does this mean that the device is different from the one mentioned in https://github.com/Koenkk/zigbee2mqtt/discussions/19760?

I am pretty new to this. I tried to add a new unsupported device, but my Zigbee2MQTT client was not able to start after adding the JS file. I followed the instructions on https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html .  
Did I make a mistake?

```
const fz = require('zigbee-herdsman-converters/converters/fromZigbee');
const tz = require('zigbee-herdsman-converters/converters/toZigbee');
const exposes = require('zigbee-herdsman-converters/lib/exposes');
const reporting = require('zigbee-herdsman-converters/lib/reporting');
const extend = require('zigbee-herdsman-converters/lib/extend');
const e = exposes.presets;
const ea = exposes.access;

const definition = {
    zigbeeModel: ['TRADFRI_bulb_GU10_WW_345lm'], // The model ID from: Device with modelID 'lumi.sens' is not supported.
    model: 'LED2106R3', // Vendor model number, look on the device for a model number
    vendor: 'IKEA', // Vendor of the device (only used for documentation and startup logging)
    description: 'TRADFRI LED bulb GU10 345 lumen, dimmable, white spectrum', // Description of the device, copy from vendor site. (only used for documentation and startup logging)
    fromZigbee: [], // We will add this later
    toZigbee: [], // Should be empty, unless device can be controlled (e.g. lights, switches).
    exposes: [], // Defines what this device exposes, used for e.g. Home Assistant discovery and in the frontend
    extend: extend.light_onoff_brightness_colortemp(),

};

module.exports = definition;
```

**Reply by Koenkk** [reply]
The model ID was wrong in https://github.com/Koenkk/zigbee2mqtt/discussions/19760, should be `TRADFRI bulb GU10 WW 345lm` instead of `TRADFRI_bulb_GU10_WS_345lm` which I fixed now.
